tests/Controller/EnMarche/BoardMemberControllerTest.php

```diff
@@ -16,7 +17,8 @@
  */
 class BoardMemberControllerTest extends WebTestCase
 {
-    use ControllerTestTrait;
+    use ControllerTestTrait,
```

Comment:
Same here for the declaration and also in :
-  `MembershipControllerTest`
-  `AdherentSecurityControllerTest`
- `MailTestCase`

Reply:
done